TESTING.md

```diff
@@ -431,6 +436,53 @@ To open the context menu:
 12. **Verify**: You get an answer similar to:
     > Squirrels are small, bushy-tailed rodents that are found all over the world. Here are some key facts about squirrels...
 
+## Model dropdown
+
+### Pro account
+
+#### Chat
+1. Login to Cody Pro account
+2. Create new chat
+3. User is able to change default LLM
+4. Change model to ChatGPT 4
+5. Send message 
+    > What model are you?
+6. User should get the response that model is ChatGPT
+7. Change account
```

Comment:
I would consider a bit different wording to avoid confusion.
Maybe just:
`7. Change account to different one and then back to your pro account`
?